pipelines.adoc

```diff
@@ -9,21 +9,47 @@ permalink: /guides/pipelines-getting-started/
+Kabanero uses link:https://github.com/tektoncd/pipeline/tree/master/docs#usage[Tekton pipelines] to illustrate a continuous input and continuous delivery (CI/CD) workflow. Kabanero provides a set of default tasks and pipelines that can be associated with application stacks.  These pipelines validate the application stack is active, build the application stack, publish the image to a container registry, and then deploys the application to the Kubernetes cluster. You can also create your own tasks and pipelines and customize the pre-built pipelines and tasks. All tasks and pipelines are activated by  link:https://github.com/kabanero-io/kabanero-operator[Kabanero's standard Kubernetes operator].
 
 To learn more about pipelines and creating new tasks, see link:https://github.com/tektoncd/pipeline/blob/master/docs/tutorial.md[the pipeline tutorial].
 
-== Tasks and Pipelines
+== Default Tasks and Pipelines
 
-Each application stack contains a set of default tasks and pipelines that are created when the application stack is built. The application stack build process copies the task and pipeline files from link:https://github.com/kabanero-io/collections/tree/master/incubator/common/pipelines/default[the application stack repo] to your `/pipelines` directory.
+The default Kabanero tasks and pipelines are provided in the link:https://github.com/kabanero-io/kabanero-pipelines/tree/master/pipelines/incubator[Kabanero pipelines repository].  These can be associated with an application stack in the Kabanero custom resource definition.  An example is provided below:
+
+```
+apiVersion: kabanero.io/v1alpha1
+kind: Kabanero
+metadata:
+  name: kabanero
+spec:
+  version: "0.6.0"
+  stacks: 
+    repositories: 
+    - name: central
+      https:
+        url: https://github.com/kabanero-io/collections/releases/download/0.5.0/kabanero-index.yaml
+    pipelines:
+    - id: default
+      sha256: 14d59b7ebae113c18fb815c2ccfd8a846c5fbf91d926ae92e0017ca5caf67c95
+      https:
+        url: https://github.com/kabanero-io/kabanero-pipelines/releases/download/0.6.0/default-kabanero-pipelines.tar.gz
+```
+
+When the Kabanero operator activates the CRD, it will associate the pipelines in the pipelines archive with each of the stacks in the stack hub.  The default pipelines are intended to work with all the stacks in the stack hub in the example above. All of the pipeline related resources like the tasks, trigger bindings, and pipelines prefix the name of the resource with the keyword `StackId`.  When the operator activates these, it will replace the keyword with the name of the stack it is activating.
```

Comment:
Change
it will associate
to
it associates

Change
in the example above.
to
in the previous example.

Change
pipeline related
to
pipeline-related

Change
resources like the tasks, trigger bindings, and pipelines prefix
to
resources (the tasks, trigger bindings, and pipelines)
or
resources (such as the tasks, trigger bindings, and pipelines)

Change
activates these, it will replace the keyword
to
activates these resources, it replaces the keyword